### DISCUSSION TITLE
State name

### DISCUSSION THREAD

**Discussion by nicoleliu0046** [synthetichealth/synthea#1030] [General]
I  wanted to export a number of patients data in different states, but failed when the states' names have more than one word. I followed the website, entered code run_synthea -p 100 "New York" --exporter.csv.export true. But it says there is an error before "New". How can I solve that? 
Btw, I want to generate both csv file and json file for the exactly same patients. How should I enter the code? I tried adding both "--exporter.csv.export true" and " --exporter.json.export true" at the end of my code, but it only export one type of file.
Thank you!

**Comment by jawalonoski** [comment]
On Mac or Linux,

```
./run_synthea -p 100 --exporter.csv.export=true "New York"
```

On Windows, you probably have to escape the space in New York or something weird.
